stdlib/platform/java/arraylist.wyv

```diff
@@ -0,0 +1,33 @@
+module def arraylist(java : Java)
+import java:wyvern.stdlib.support.ArrayListWrapper.arraylist
+
+// private?
```

Comment:
If you want to make DynArrayList private, which is probably a good idea, the way to do it is to ascribe a type to the module.  Just put ": T" at the end of the module def line, for some type T.

I'm adding a feature request related to "private"
In this case, you could define type Tarraylist (we need a naming convention for this, this is just one suggestion, maybe you have a better one) in a file Tarraylist.wyt as the public signature for arraylist.